### PR TITLE
feat: added spacer options for forgotPasswordForm

### PR DESCRIPTION
**Pull request by mighttymike**

### CODE REVIEW

File: lib/src/config/forgot_password_spacer_options.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class ForgotPasswordSpacerOptions {
+  const ForgotPasswordSpacerOptions({
```
Comment (freekvandeven) [inline lib/src/config/forgot_password_spacer_options.dart:5]: where are the /// comments for the constructor and the ForgotPasswordSpacerOptions class?

File: lib/src/config/login_options.dart [hunk 2]
```diff
@@ -61,6 +63,7 @@ class LoginOptions {
   /// Option to modify the spacing between the title, subtitle, image, form,
   /// and button.
   final LoginSpacerOptions spacers;
+  final ForgotPasswordSpacerOptions forgotPasswordSpacerOptions;
```
Comment (freekvandeven) [inline lib/src/config/login_options.dart:66]: Where is the /// comment explaining what this is?

File: lib/src/widgets/forgot_password_form.dart [hunk 3]
```diff
@@ -78,32 +78,63 @@ class _ForgotPasswordFormState extends State<ForgotPasswordForm> {
...
-                    focusNode: _focusNode,
-                    onChanged: _updateCurrentEmail,
-                    validator: widget.options.validations.validateEmail,
-                    initialValue: options.initialEmail,
-                    keyboardType: TextInputType.emailAddress,
-                    textInputAction: TextInputAction.next,
-                    style: options.emailTextStyle,
-                    decoration: options.emailDecoration,
-                  ),
+                child: Column(
```
Comment (freekvandeven) [inline lib/src/widgets/forgot_password_form.dart:113]: what is the use of this column if there is only 1 child?

**Review by freekvandeven — APPROVED** [review]

**Review by Gorter-dev — APPROVED** [review]

**Comment by Gorter-dev** [comment]
In the future use present tense in the commit message

**Pull request merged**